### PR TITLE
Disable the FQDN lookup by default.

### PR DESCRIPTION
**Pull request by bmoyles0117**

### CODE REVIEW

File: collectd.conf.tmpl [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Interval 60
+FQDNLookup false
```
Comment (igorpeshansky) [inline collectd.conf.tmpl]: Might want to add a comment that the FQDN is not used by the Stackdriver monitoring agent.

**Comment by igorpeshansky** [comment]
This shouldn't affect the agent behavior at all, except remove spurious errors when (reverse) DNS is not available.

File: collectd.conf.tmpl (hunk 1 above)
Comment (bmoyles0117) [reply collectd.conf.tmpl]: PTAL.

**Comment by dhrupadb** [comment]
I believe the idea is to cherry pick the commit into those branches instead of explicitly adding them.

**Comment by bmoyles0117** [comment]
The legacy branches have a different file name, so I'll just create another branch once this one goes through and paste the contents into the right place. SG?

**Comment by dhrupadb** [comment]
You should still be able to cherry pick it in I think. I'll let igorp@ confirm that.

File: collectd.conf.tmpl [hunk 2]
```diff
@@ -1,5 +1,8 @@
 Interval 60
 
+# The Stackdriver Agent does not rely on FQDNLookup to determine the hostname.
```
Comment (igorpeshansky) [inline collectd.conf.tmpl]: `The Stackdriver agent does not use fully qualified domain names.`

**Comment by igorpeshansky** [comment]
Yes, you cherry-pick, let the merge fail, resolve the conflict (possibly by copy/pasting the content), and commit. That way the commit is associated with the original. Then you create a PR that references this one.

File: collectd.conf.tmpl (hunk 2 above)
Comment (bmoyles0117) [reply collectd.conf.tmpl]: PTAL

**Pull request merged**